api/src/main/java/io/opentelemetry/trace/SpanContext.java

```diff
@@ -30,19 +30,27 @@
  */
 @Immutable
 public final class SpanContext {
-  private static final Tracestate TRACESTATE_DEFAULT = Tracestate.builder().build();
   private final TraceId traceId;
   private final SpanId spanId;
   private final TraceOptions traceOptions;
   private final Tracestate tracestate;
 
   /**
-   * A blank {@code SpanContext} that can be used for no-op operations.
+   * Returns the blank {@code SpanContext} that can be used for no-op operations.
    *
+   * @return the blank {@code SpanContext}.
    * @since 0.1.0
    */
-  static final SpanContext BLANK =
-      new SpanContext(TraceId.INVALID, SpanId.INVALID, TraceOptions.DEFAULT, TRACESTATE_DEFAULT);
+  public static SpanContext getBlank() {
+    return BLANK;
+  }
+
+  private static final SpanContext BLANK =
```

Comment:
nit: Move this at the top of the class :)

Reply:
Fixed.